params/config.go

```diff
@@ -343,14 +345,14 @@ func (c *CliqueConfig) String() string {
 
 // BorConfig is the consensus engine configs for Matic bor based sealing.
 type BorConfig struct {
-	Period                uint64 `json:"period"`                // Number of seconds between blocks to enforce
-	ProducerDelay         uint64 `json:"producerDelay"`         // Number of seconds delay between two producer interval
-	Sprint                uint64 `json:"sprint"`                // Epoch length to proposer
-	BackupMultiplier      uint64 `json:"backupMultiplier"`      // Backup multiplier to determine the wiggle time
-	ValidatorContract     string `json:"validatorContract"`     // Validator set contract
-	StateReceiverContract string `json:"stateReceiverContract"` // State receiver contract
-
-	OverrideStateSyncRecords map[string]int `json:"overrideStateSyncRecords"` // override state records count
+	Period                   map[string]uint64 `json:"period"`                   // Number of seconds between blocks to enforce
+	ProducerDelay            uint64            `json:"producerDelay"`            // Number of seconds delay between two producer interval
+	Sprint                   uint64            `json:"sprint"`                   // Epoch length to proposer
+	BackupMultiplier         map[string]uint64 `json:"backupMultiplier"`         // Backup multiplier to determine the wiggle time
+	ValidatorContract        string            `json:"validatorContract"`        // Validator set contract
+	StateReceiverContract    string            `json:"stateReceiverContract"`    // State receiver contract
+	OverrideStateSyncRecords map[string]int    `json:"overrideStateSyncRecords"` // override state records count
+	BurntContract            map[string]string `json:"burntContract"`            // governance contract where the token will be sent to and burnt in london fork
```

Comment:
Why do we need all these maps?

Reply:
Could you make "BurntContract" a types.Address? Otherwise, you have to do string -> types.Address every time you want to use it.

Reply:
OverrideStateSyncRecords -  this is due to one of the chain issues which we fixed sometime back
BurntContract, BackupMultiplier, Period - this is based on blocks. if tomorrow from a particular block we want to change any of these values, we can do it without affecting the chain

Since previous contracts type is string, it will good to have consistency across burnContract as well.